ui/src/components/AdvancedSettingsModal.vue

```diff
@@ -187,6 +187,28 @@
 
           <hr class="advanced-settings-divider" />
 
+          <div>
+            <div class="d-flex justify-content-between align-items-center">
+              <div class="w-75">
+                <label class="mb-0" for="rest">
+                  <p class="font-weight-bold mb-0">Enable REST</p>
+                </label>
+              </div>
+              <div>
+                <toggle-switch
+                  id="rest"
+                  class="align-self-center"
+                  :on="settings.rest"
+                  @toggle="status => (settings.rest = rest)"
+                ></toggle-switch>
+              </div>
+            </div>
+            <small class="w-sm-75 d-block text-muted mt-1">
+              Enable public REST endpoints.
```

Comment:
Note to self: we should add additional text here to warn about privacy/DoS risks when enabling the REST API. Happy to merge this PR without that wording and I will add before next update, since we are going to add additional configuration options in the next update and change some wording of the existing options.